### DISCUSSION TITLE
Problem linking Angle

### DISCUSSION THREAD

**Discussion by timluther** [microsoft/vcpkg#19019] [Q&A]
Hi,
  I've just cloned a project on to a new windows machine but I'm getting problems linking against angle. I'm using static linking and all the libraries are in the triplet x64-windows-static. Everything else seems to link fine but I'm getting many link errors such as:
  
Error	LNK2019	unresolved external symbol __imp_glBindBuffer referenced in function ...

the '__imp_' prefix suggests its looking for a dll lib, which I suppose is fine as long as that's linking against the static standard libs but either way, visual studio reports this as a linking error. I've had similar problems with a few other libraries (libcurl and an encryption library needed by minizip-ng)  and I've #def'd out support for those for the time being: I can't currently do without Angle however.

My understanding is that vcpkg linking should 'just work' without needing explicit linkages. Does anybody know what might be causing these linker errors?

**Comment by timluther** [comment]
The problem only occurs with static linking leading me to assume that Angle cannot be statically linked.

**Reply by BillyONeal** [reply]
It is true that not all ports respect all settings :(
